src/ihx/EvalEngine.hx

```diff
@@ -69,6 +70,7 @@ class EvalEngine
         var args = switch evalMode {
             case neko: ["-neko", tmpBuildPath, "-cp", tmpDir, "-main", tmpHxFname, "-cmd", "neko "+tmpBuildPath];
             case hashlink: ["-hl", tmpBuildPath, "-cp", tmpDir, "-main", tmpHxFname, "-cmd", "hl "+tmpBuildPath];
+            case python: ["-python", tmpBuildPath, "-cp", tmpDir, "-main", tmpHxFname, "-cmd", "python "+tmpBuildPath];
```

Comment:
this didn't work on ubuntu because haxe targets python 3 which must be run with the executable "python3". is it fine if I change this to `... "-cmd", "python3 "+tmpBuildPath];`?